db/migrate/20190821224909_rename_ordered_at_column_on_orders.rb

```diff
@@ -0,0 +1,5 @@
+class RenameOrderedAtColumnOnOrders < ActiveRecord::Migration[5.2]
+  def change
+    rename_column :orders, :ordered_at, :order_request_sent_at
```

Comment:
We should be able to compress all the changes into a single migration.

Reply:
I toyed with combining them, but ultimately separated them to attempt to make the migration names more concise and descriptive. Otherwise it would be "RenameOrderedAtColumnOnOrdersAndOrderItems" which felt a bit wordy but I guess it's not really much better than having "RenameOrderedAtColumnOnOrders" and "RenameOrderedAtColumnOnOrderItems". lol 🤷‍♂ 

We didn't have any "rename" precedent so I went with separate migrations, but I'll switch it to one.